src/core/zowe/core_for_zowe_sdk/validators.py

```diff
@@ -12,9 +12,12 @@
 
 import commentjson
 from jsonschema import validate
+import os
+from typing import Union, Optional
+import requests
 
 
-def validate_config_json(path_config_json: str, path_schema_json: str):
+def validate_config_json(path_config_json: Union[str, dict], path_schema_json: str, cwd: str):
```

Comment:
curious if we should make `cwd` optional and fallback to `$PWD` 😋 

Either way, this new option need to be added as part of the parameter list below 😋